bin/endpoint.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python3
```

Comment:
Not really sure that the name `endpoint` is the best name for this file... Maybe `endpoint_db` or `boss_rds` or something to show that the file works with the database, not the EC2 instance

Reply:
Ok sounds good. Wanted to draw that parallel with the vault instance. But it IS true that the vault script interact with the hashicorp vault commands themselves. I also thought maybe in the future this script could be expanded to maybe interacting with other parts of the endpoint instance if needed? But I think it's probably more accurate to rename it to endpoint_db. Thanks